### ISSUE TITLE
Lack of support for notebooks

### ISSUE THREAD

**Issue by rafael-pinheiro** [pgaskin/NickelMenu#117]
Kobo Sage and Kobo Elipsa have a new feature: Notebooks.

Currently there is no way to open a notebook or list of notebooks via NickelMenu. There is also no way of adding an item to the notes app menu as far as I can see.

Is that something that is in the radar? Any plans for bringing that functionality in?

**Comment by pgaskin** [comment]
Yes, that's planned in #109. I just haven't had enough time to implement it yet.

**Comment by pgaskin** [comment]
duplicate of #109

**Comment by Pololot64** [comment]
I am willing to help try implementing this though I don't have any experience and cannot reverse engineer myself.

**Comment by Pololot64** [comment]
I did a quick read of the source and extracted the latest kobo firmware and found: 

```_ZN15LibraryNavMixin13showNotebooksE14MainNavSection```

I think this may be the relevant section. I can make a pr and try to test

**Comment by Pololot64** [comment]
I have tested and that string does indeed launch the notepad. However, notes created cannot be opened for me...

**Comment by Pololot64** [comment]
Also, I was doing some more testing and realized that my kobo Libra 2 has full support for my laptop stylus. Pressure support and everything (eraser and highlighter buttons). This may mean that full notebook support can be unlocked for this device as well.

**Comment by Pololot64** [comment]
https://github.com/pgaskin/NickelMenu/pull/144
